readme.md

```diff
@@ -82,264 +101,296 @@ var civic = {
+    this.fuel = 100;
+  }
+}
 
-Constructor functions are a lot like the `makeCar` function we just created, but
-they're supported by JS and we use the `new` keyword to use the constructor
-function.
-<!--maybe demonstrate that constructor is a property, All objects inherit a constructor property from their prototype:-->
+const celica = new Car("Toy-Yoda Celica", "limegreen");
+const civic = new Car("Honda Civic", "lemonchiffon");
+```
 
-Note that constructor functions start with a capital letter to make it obvious
-that they are constructors. This isn't necessary, but is a convention and you
-should follow it!
+Classes are a lot like the `makeCar` function we just created, but they're supported by JS and we use the `new` keyword to generate instances of an object (just like our earlier `celica` and `civic` examples).
 
-## Break (10 min)
+> Note that classes start with a capital letter to make it obvious
+that they are constructors. This isn't necessary, but is a convention you should follow.
 
-### I-Do: Make a Person Constructor Function (10 min)
+### I Do: Make a Person Class (10 minutes / 0:45)
 
 ```js
-function Person(initialName) {
-  this.name = initialName;
-  this.species = "Homo Sapiens";
-  this.speak = function() {
-    return "Hello! I'm " + this.name;
-  };
+class Person {
+  // We use the constructor method to initialize properties for a class instance.
+  // It takes whatever arguments we want to pass into an instance.
+  constructor(initialName){
+    this.name = initialName;
+    this.species = "Homo Sapiens";
+  }
+  // We define any methods accessible to an instance outside of the constructor
+  speak(){
+    return `Hello! I'm ${this.name}`;
+  }
 }
 
-var adam = new Person("Adam");
-var bob =  new Person("Bob");
-
-adam.name // "Adam"
-adam.speak() // "Hello! I'm Adam"
-
-bob.name // "Bob"
-bob.speak() // "Hello! I'm Bob"
+var adrian = new Person("Adrian");
+adrian.speak(); // "Hello, I'm Adrian"
 ```
 
-Notice the use of `this`, and the fact that we don't return anything. Here's why we write constructor functions this way:
+#### `this`
 
-When we run a constructor function with `new`, Javascript will automatically:
+Notice the use of `this`, and the fact that we don't return from the class. Here's why we write classes this way...
 
-1. Create an new, empty object for us.
-2. Call the constructor function on that object (`this` -> the new object)
-3. Return the object
+When we generate a class instance using `new`, Javascript will automatically...
+  1. Create an new, empty object for us  
+  2. Generate a context for that object (`this` -> the new object)  
+  3. Return the object  
 
-### You-Do Exercise: Make a Car Constructor Function (10 min)
+#### Where are the Commas?
 
-Write a constructor function to replace our `makeCar` function earlier.
+Unlike object notation, you do not need to use commas when separating class methods.
 
-See car.js for a solution.
+### You Do: Make an ATM Class (20 minutes / 1:05)
```

Comment:
can you be more specific w/ specs here? .e.g

It takes 1 argument, initial balance.

It has 2 methods, withdraw and deposit. ...